### PR TITLE
feature: Novo Global CSS

### PR DESCRIPTION
**Pull request by icirellik**
Load global layout css that many novo components depend on.

### CODE REVIEW

File: src/novo/src/index.ejs [hunk 1]
```diff
@@ -4,15 +4,7 @@
...
 
-  <% if (env === 'production') { %>
-  <link rel="preload" href="<%= manifest.artsy %>" as="script">
-  <link rel="preload" href="<%= manifest.artsyCommon %>" as="script">
-  <link rel="preload" href="<%= manifest.artsyNovo %>" as="script">
-  <link rel="preload" href="<%= manifest.common %>" as="script">
-  <link rel="preload" href="<%= manifest.commonReact %>" as="script">
-  <link rel="preload" href="<%= manifest.commonUtility %>" as="script">
-  <link rel="preload" href="<%= manifest.runtime %>" as="script">
-  <% } %>
```
Comment (icirellik) [inline src/novo/src/index.ejs:15]: Preloads are not required for these files.

**Review by eessex — APPROVED** [review]
This looks functional to me 👍
I do wonder why we need to include these styles? My hope is that the collect page or blank page would work without them, potentially we could drop all standard css in the new template.

**Comment by icirellik** [comment]
@eve it's definitely worth ticketing and converting them to styled-components in the future.

**Comment by eessex** [comment]
@icirellik i wonder if you have examples of components that are broken without these styles in the current pages?

**Pull request merged**